studio/components/interfaces/Database/Hooks/CreateHook/ServiceConfigForm/InputServiceUrl.tsx

```diff
@@ -17,7 +17,7 @@ const InputServiceUrl: FC = observer(({}) => {
       onChange={(e) =>
         _localState.onFormChange({
           key: 'serviceUrl',
-          value: e.target.value,
+          value: e.target.value.replaceAll(' ', ''),
```

Comment:
we shouldn't be stripping in `onChange` i think - we should just do this prior to saving
also, i'd reckon we just do a `trim` though. most of the fields in the dashboard where we remove whitespace is using trim, so just to keep things consistent.

Reply:
ahh okay yeah fair enough, not supposed to have spaces in the middle of URLs